packages/flutter/test/rendering/proxy_box_test.dart

```diff
@@ -510,3 +522,18 @@ class _TestPathClipper extends CustomClipper<Path> {
   @override
   bool shouldReclip(_TestPathClipper oldClipper) => false;
 }
+
+class _TestMarkNeedsSemanticsUpdateCountingRenderFractionalTranslation extends RenderFractionalTranslation {
```

Comment:
Maybe rename to `_TestSemanticsUpdateRenderFractionalTranslation` for brevity?

Reply:
Done